BREEZE.rst

```diff
@@ -2056,6 +2065,28 @@ This is the current syntax for  `./breeze <./breeze>`_:
 
   Flags:
 
+  -S, --version-suffix-for-pypi SUFFIX
+          Adds optional suffix to the version in the generated backport package. It can be used
```

Comment:
Is this only for backports or regular providers too?

Reply:
For regular providers too